### DISCUSSION TITLE
Is it possible to add more than one custom action, if so what's the correct syntax for sftpgo.json?

### DISCUSSION THREAD

**Discussion by arthurschaap** [drakkan/sftpgo#365] [Q&A] [answered]
"actions": {
      "execute_on": ["upload"],
      "hook": "/path/to/customscript"},
      "execute_on": ["pre-delete"],
      "hook": "/path/to/customscript"},

is not working. Tried several combinations, but did not find the right one.

**Comment by drakkan** [comment, marked as answer]
Hi,

please try something like this:

```
"execute_on": ["upload","pre-delete"]
```

then within your hook you can get the [action](https://github.com/drakkan/sftpgo/blob/main/docs/custom-actions.md) and do different things based on it, if that's what you mean

**Comment by arthurschaap** [comment]
Thanks! That will do the trick.
